libs/ledger-live-common/src/families/tezos/signOperation.ts

```diff
@@ -32,53 +32,81 @@ export const signOperation: SignOperationFnSignature<Transaction> = ({
             false,
             DerivationType.ED25519,
           );
+
           tezos.setProvider({ signer: ledgerSigner });
```

Comment:
Is this still necessary, now that we call `ledgerSigner.sign` directly?

Reply:
Seems like yes: https://github.com/LedgerHQ/ledger-live/pull/6088#issuecomment-1934713512